packages/paste-core/components/toast/src/types.ts

```diff
@@ -18,6 +18,14 @@ export interface ToastProps extends React.HTMLAttributes<HTMLDivElement>, Pick<B
    * Use this to set focus within the toast when it is rendered
    */
   setFocus?: boolean;
+  /**
+   * These labels support i18n
+   */
+  i18nDismissLabel?: string;
+  i18nErrorLabel?: string;
+  i18nNeutralLabel?: string;
+  i18nSuccessLabel?: string;
+  i18nWarningLabel?: string;
```

Comment:
suggestion: it might seem redundant, but each one of these having their own tsdoc comment would be beneficial. These provide comments in VS code when you hover over the prop to let you know what the prop is for. Right now the last 4 wouldn't get a VS Code hint, but `i18nDismissLabel` would say "These labels support i18n", which in isolation might not feel very useful to the developer using it.

For example, hovering over `onDismiss` 
![image](https://user-images.githubusercontent.com/368249/155599763-d6fc9b03-4b2a-4dd7-b245-8307d490dc55.png)

Reply:
ooo thanks for the catch on this; we don't employ it very often that I've seen in our types so I'm glad I know more about it now. It's been implemented! I used the descriptions from the anatomy table in the docs.